src/msi_trilinos.cc

```diff
@@ -20,7 +20,7 @@
 #include <Epetra_CrsMatrix.h>
 
 // ***********************************
-// 		MSI_LINEAR SYSTEM
+//    MSI_LINEAR SYSTEM
```

Comment:
This file appears to only have whitespace changes. If formatting needs to be fixed (tabs to two spaces, etc.) then please submit a separate PR with commits that specifically address it.